content/docs/command-reference/list-url.md

```diff
@@ -0,0 +1,97 @@
+# list-url
+
+> Aliased to `dvc ls-url`
+
+List contents from a supported URL (for example `s3://`, `ssh://`, and other
+protocols).
+
+> Useful to find data to `dvc get-url` or `dvc import-url`.
```

Comment:
I'd link back from those other refs to this one too.

Feel free to copy the notes from the ref pages of `get` and `import` (which mention `list`).